adobecom/homepage
# MWPW-139086 - Do not preload IMS

* Only preload IMS if there are traces of a previous session
* Scripts.js cleanup to better follow milo standards
* Fix linting being broken

Resolves: [MWPW-139086](https://jira.corp.adobe.com/browse/MWPW-139086)

## Test URLs:
- Before: https://main--homepage--adobecom.hlx.page/homepage/loggedout-cm?martech=off
- After: https://ims--homepage--adobecom.hlx.page/homepage/loggedout-cm?martech=off

## Steps to test (anonymous scenario):
1. Visit the before & after page above.
2. Before - IMS loads while milo is loading pre-LCP code (~9th position)
3. After - IMS loads as part of the normal Milo lifecycle. (much lower position)

## Steps to test (login scenario):
1. Sign in on IMS branch
2. Observe: IMS is loaded higher in the waterfall

## Other notes:
This de-emphasizes cross-domain login IMS fast-tracking. If a visitor logs into something like business.adobe.com first, the full milo stack will load because the sessionStorage check will not find traces of a previous session for the current www domain. I view this as an acceptable trade off  because:

1. Most visitors do not cross domains 
2. They are still taken to the final destination
3. There's already flickering of experiences today given how loadArea is currently run.
